specifications/api/swagger.yaml

```diff
@@ -4460,10 +4460,25 @@ x-errNotEnoughMoney: &errNotEnoughMoney
   properties:
     message:
       type: string
-      description: May occur when there's not enough money in the wallet to cover a requested payment.
+      description: |
+        Indicates that there's not enough money in the wallet to cover a
+        requested payment.
```

Comment:
Add a sentence that describes the `shortfall` field in more detail.
Emphasise that the shortfall just describes the assets that are missing, and that if supplied, would unblock whatever operation it was that the user was attempting.

Reply:
Fixed in 0432db5cad7de78690601323617959a54ac12cea.